foreman/data_refinery_foreman/gatherer/management/commands/gather_accessions.py

```diff
@@ -217,28 +201,52 @@ def handle(self, *args, **options):
                 continue
             agents.append(cls(options))
 
-        entries = set()
+        gathered_accessions = set()
         for agent in agents:
-            entries.update(agent.collect_data())
+            agent_accessions = agent.collect_data()
+            agent_accessions_count = len(agent_accessions)
+
+            gathered_accessions.update(agent_accessions)
 
-        entries = sorted(  # Sort the resulting list.
-            (entry for entry in entries if self.RE_ACCESSION.match(entry.accession_code)),
-            key=lambda entry: (
-                self.RE_ACCESSION.match(entry.accession_code).group(1),
-                int(self.RE_ACCESSION.match(entry.accession_code).group(2)),
+            logger.info(
+                f"Since {options['since']} {agent} has gathered {agent_accessions_count} "
+                f"new accession{pluralize(agent_accessions_count)}."
+            )
+
+        gathered_accessions = sorted(  # Sort the resulting list.
+            (ga for ga in gathered_accessions if self.RE_ACCESSION.match(ga.accession_code)),
+            key=lambda ga: (
+                self.RE_ACCESSION.match(ga.accession_code).group(1),
```

Comment:
Could you add a comment where `RE_ACCESSION` is defined so it is easier to understand.